config/initializers/delayed_job_config.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Delayed::Worker.destroy_failed_jobs = false
+Delayed::Worker.sleep_delay = 60
+Delayed::Worker.max_attempts = 3
+Delayed::Worker.max_run_time = 5.minutes
+Delayed::Worker.read_ahead = 10
+Delayed::Worker.default_queue_name = 'default'
+Delayed::Worker.delay_jobs = !Rails.env.test?
+Delayed::Worker.raise_signal_exceptions = :term
+Delayed::Worker.logger = Logger.new(Rails.root.join('log/delayed_job.log'))
```

Comment:
Unconditionally logging to a file here is problematic… This should use similar logic to the normal rails logging initializer. For use in docker, it should be logging to stdout/stderr?